apps/site/assets/css/_schedule-page-line-diagram.scss

```diff
@@ -141,6 +141,14 @@ $line-width: 8px; // width of the colored line in the diagram
   }
 }
 
+.m-schedule-table-crowding {
```

Comment:
Is this class needed, as opposed to putting this rule on `.m-schedule-diagram__prediction-crowding`?

Reply:
You're right, it's a bit superfluous.